instrumentation/rmi/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/rmi/context/ContextPayload.java

```diff
@@ -42,9 +34,8 @@ public ContextPayload(Map<String, String> context) {
     this.context = context;
   }
 
-  public static ContextPayload from(Span span) {
+  public static ContextPayload from(Context context) {
     ContextPayload payload = new ContextPayload();
-    Context context = Context.current().with(span);
     GlobalOpenTelemetry.getPropagators().getTextMapPropagator().inject(context, payload, SETTER);
```

Comment:
there's still a global call in here...that seems like a problem.

Reply:
Yep - currently all `inject()` calls are using global propagators. That's going to be the subject for my next PR 😄